projects/packages/sync/src/class-actions.php

```diff
@@ -304,7 +304,7 @@ public static function sync_allowed() {
 			return false;
 		}
 
-		if ( ( new Status() )->is_staging_site() ) {
+		if ( ( new Status() )->in_safe_mode() ) {
```

Comment:
It's probably been discussed already, but is it ok to suddenly enable Sync for those former "staging" sites?

Reply:
Since we allowed Safe mode on all "staging" sites way back I don't think there will be much difference. Well, there will be more requests, but that's all. No new IDCs should happen, just more volume.
On the other hand, all sites with safe mode enabled and a fresh connection are now stuck with broken sync for no good reason.

Reply:
This is the PR that I was referring to that allows SM on staging sites: https://github.com/Automattic/jetpack/pull/23053

Reply:
Thanks for the explanation.
You're right, that should be fine.